core/src/test/java/org/apache/brooklyn/core/mgmt/persist/FileBasedStoreObjectAccessorWriterTest.java

```diff
@@ -61,6 +76,30 @@ public void testFilePermissions600() throws Exception {
         FileBasedObjectStoreTest.assertFilePermission600(file);
     }
 
+    // Fails ~3 times on 5000 runs in Virtualbox (Ubuntu Xenial).
```

Comment:
Well it would certainly be nice to have an explanation of why it's happening. I am still not ready to attribute it to VB flakiness.